app/queries/get_application_choices_for_providers.rb

```diff
@@ -20,6 +20,6 @@ def self.call(providers:)
       .where('courses.provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.visible_years)
       .or(ApplicationChoice.includes(*includes)
         .where('courses.accredited_provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.visible_years))
-      .where('status IN (?)', ApplicationStateChange.states_visible_to_provider).includes([:accredited_provider])
+      .where('status IN (?)', ApplicationStateChange.states_visible_to_provider_in_vendor_api).includes([:accredited_provider])
```

Comment:
Out of interest, what's the reason for not showing deferred offers via the API?

Reply:
It's an unexpected state, so potentially a breaking change